_config.yml

```diff
@@ -29,7 +29,3 @@ defaults:
       type: components
     values:
       sidenav: components
-  - scope:
```

Comment:
Can we remove lines 19 through 21 above, which still reference the research collection?

Reply:
Done bfed82f229dabcb0aad2a5c59d0d22faf13830f4